aws/table_aws_cloudformation_stack_resource.go

```diff
@@ -0,0 +1,223 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/cloudformation"
+	"github.com/aws/aws-sdk-go-v2/service/cloudformation/types"
+
+	cloudformationv1 "github.com/aws/aws-sdk-go/service/cloudformation"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsCloudFormationStackResource(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_cloudformation_stack_resource_resource",
```

Comment:
Running 'steampipe plugin list' with this PR's code, it prints a warning "table 'aws_cloudformation_stack_resource' has inconsistent Name property: 'aws_cloudformation_stack_resource_resource'". Running a query fails with "ERROR: relation "dev_charlie.aws_cloudformation_stack_resource" does not exist (SQLSTATE 42P01)".
If I delete the duplicate "_resource" at the end of the name, and rebuild the plugin, both commands work fine.

Reply:
Ahh @stevebutler, It was my bad. Updated the table name to `aws_cloudformation_stack_resource`. Thanks!